File: Python/tests/test_bandit4arm_lapse_decay.py
import pytest

from hbayesdm.models import bandit4arm_lapse_decay


def test_bandit4arm_lapse_decay():
    _ = bandit4arm_lapse_decay(example=True, niter=2, nwarmup=1, nchain=1, ncore=1)


if __name__ == '__main__':
    pytest.main()

File: R/tests/testthat/test_bandit4arm_lapse_decay.R
context("Test bandit4arm_lapse_decay")
library(hBayesDM)

test_that("Test bandit4arm_lapse_decay", {
  # Do not run this test on CRAN
  skip_on_cran()

  expect_output(bandit4arm_lapse_decay(
      data = "example", niter = 10, nwarmup = 5, nchain = 1, ncore = 1))
})
